### PR TITLE
Add documentation about fastest

### PR DESCRIPTION
**Pull request by alexislefebvre**

### CODE REVIEW

**Comment by alexislefebvre** [comment]
I found fastest really easy to use, compared to paratest.

File: fastest.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Introduction
+============
+
+[fastest](https://github.com/liuggio/fastest) is a tool for executing PHPUnit tests in parallel.
+Once LiipFunctionalTestBundle has been configured, it's easy to use fastest.
+
+Installation
+============
+
+1. Install paratest: `composer require "liuggio/fastest=~1.4"`
```
Comment (andrerom) [inline fastest.md]: Paratest?

Comment (alexislefebvre) [inline fastest.md]: @andrerom This documennt page is about the fastest tool.

Comment (alexislefebvre) [inline fastest.md]: @andrerom Oops, thanks for reporting the copy-paste error.

Comment (alexislefebvre) [inline fastest.md]: It has been fixed after this PR was merged: https://github.com/liip/LiipFunctionalTestBundle/pull/283

**Pull request merged**